src/endpoints/accounts-v2/entities/account.ts

```diff
@@ -0,0 +1,54 @@
+import { SwaggerUtils } from "@multiversx/sdk-nestjs-common";
+import { ApiProperty } from "@nestjs/swagger";
+import { AccountAssets } from "src/common/assets/entities/account.assets";
+
+export class Account {
```

Comment:
all the entities seem to be duplicates of the v1 entities. or are there differences?